paddle/fluid/framework/dist_multi_trainer.cc

```diff
@@ -129,6 +129,7 @@ void DistMultiTrainer::InitOtherEnv(const ProgramDesc &main_program) {
     InitDumpEnv();
   }
   pull_dense_worker_->SetRootScope(root_scope_);
+  pull_dense_worker_->SetPlace(workers_[0]->place());
```

Comment:
pull dense worker has the same place as worker[0]'s, is it by design or just for easy to implement